runtime/elem/builtins/Capture.h

```diff
@@ -13,7 +13,7 @@ namespace elem
     struct CaptureNode : public GraphNode<FloatType> {
         CaptureNode(NodeId id, FloatType const sr, int const blockSize)
             : GraphNode<FloatType>::GraphNode(id, sr, blockSize),
-              ringBuffer(1, static_cast<size_t>(sr))
+              ringBuffer(1, juce::nextPowerOfTwo(static_cast<size_t>(sr)))
```

Comment:
That's the right idea, but the elem Runtime is designed to be used independently of JUCE, so we can't rely this JUCE helper here.

I have a "next power of two" function in the delay code: https://github.com/elemaudio/elementary/blob/main/runtime/elem/builtins/Delays.h#L19. Let's pull that `bitciel` into a separate helper file in `runtime/elem/builtins/helpers`, update the Delays file to include the helper, and then use that same helper here in Capture

Reply:
@nick-thompson thank you so much for reviewing.

I rebased + extracted out the `bitciel` function as you suggested. Mind taking another look and let me know if it's what you had in mind? I tested locally and it does appear to be working as expected with both `Delay` and `Capture`.

Also renamed to `bitceil` hope that's okay!